nexus/tests/config.test.toml

```diff
@@ -41,16 +41,10 @@ max_vpc_ipv4_subnet_prefix = 29
 id = "e6bff1ff-24fb-49dc-a54e-c6a350cd4d6c"
 rack_id = "c19a698f-c6f9-4a17-ae30-20d711b8f7dc"
 
-#
```

Comment:
Why drop this comment?  It still seems relevant.

Reply:
Added it back